Sources/Redis/StreamBuffer.swift

```diff
@@ -89,14 +87,12 @@ final class StreamBuffer<Stream: DuplexStream>: DuplexStream {
     }
     
     /// write bytes to the buffer stream
-    func write(_ bytes: Bytes) throws {
-        writeBuffer += bytes
+    func write(_ bytes: Bytes) throws -> Int {
+        return try write(max: bytes.count, from: bytes)
```

Comment:
This needs to be fixed. Stream buffer should still have a write buffer and a `writeBuffer: Bytes`. Otherwise it's going to be calling `libc.send` many times for each message.